client/src/parent/chores/ChoreCard.tsx

```diff
@@ -126,77 +125,137 @@ export function ChoreCard(props: Prop): JSX.Element {
     console.log("Editing due date...");
+    var updatedChore = produce(
+      props.chore,
+      (draftChore: ChoreWithAssignments) => {
+        draftChore.dueDate = date;
+      }
+    );
+
     putChoreMutation.mutate({
       url: "/api/Chores/",
-      object: props.chore,
+      object: updatedChore,
     });
-    showInfo("Editing due date..."); // TODO js (11.03.2021): Replace dummy implementation.
+    showInfo("Due date edited."); // TODO js (11.03.2021): Replace dummy implementation.
   }
 
-  function handleValueEdit() {
+  function handleValueEdit(value: number) {
     console.log("Editing amount of marbles...");
+    var updatedChore = produce(
+      props.chore,
+      (draftChore: ChoreWithAssignments) => {
+        draftChore.value = value;
+      }
+    );
+
     putChoreMutation.mutate({
       url: "/api/Chores/",
-      object: props.chore,
+      object: updatedChore,
     });
-    showInfo("Editing amount of marbles..."); // TODO js (11.03.2021): Replace dummy implementation.
+    showInfo("Amount of marbles edited."); // TODO js (11.03.2021): Replace dummy implementation.
   }
 
-  function getDescription() {
-    if (!props.chore.description) return;
-
-    return (
-      <Typography
-        className={classes.description}
-        variant="body2"
-        color="textSecondary"
-        component="p"
-      >
-        {props.chore.description}
-      </Typography>
+  function handleDescriptionEdit(description: string) {
+    console.log("Editing description...");
+    var updatedChore = produce(
+      props.chore,
+      (draftChore: ChoreWithAssignments) => {
+        draftChore.description = description;
+      }
     );
+
+    putChoreMutation.mutate({
+      url: "/api/Chores/",
+      object: updatedChore,
+    });
+    console.log("Description edited.");
+    showInfo("Description edited."); // TODO js (11.03.2021): Replace dummy implementation.
   }
 
   return (
     <Card elevation={5}>
       <BiAvatarCardHeader
-        leftAvatarLabel={props.chore.assignments.length.toString()}
-        leftAvatarNotifications={
-          props.chore.assignments.filter(
-            (assignment) =>
-              assignment.state === AssignmentState.RequestedToCheck
-          ).length
+        leftAvatarComponent={
+          <Badge
+            badgeContent={
+              props.chore.assignments.filter(
+                (assignment) =>
+                  assignment.state === AssignmentState.RequestedToCheck
+              ).length
+            }
+            color="secondary"
+          >
+            <Avatar className={classes.avatar} onClick={handleExpandClick}>
+              {props.chore.assignments.length.toString()}
+            </Avatar>
+          </Badge>
+        }
+        titleComponent={
+          <EditableText
+            text={props.chore.name}
+            editable={!cardLocked}
+            editLabel="Bezeichnung des Ämtlis"
+            validationSchema={Yup.object({
```

Comment:
@marZeu8 Hier die Validierung eines Strings mit Yup.
Die ValidationSchemas für die verschiedenen Felder könnten wir vielleicht irgendwo zentral ablegen, damit wir im Add-Dialog und beim Editieren die gleichen Regeln anwenden können. Das könnten wir vielleicht in einem späteren Schritt noch refactoren.

Reply:
Ich denke auch am Schluss werden wir wahrscheinlich so oder so noch über die Validierungen sprechen.